src/Lucene.Net.ObjectMapping/Mapping/MappedFieldResolver.QueryVisitor.cs

```diff
@@ -378,41 +378,65 @@ private static TermQuery MakeTermQuery(MappedField field, Expression term)
                 Debug.Assert(null != field, "The mapped field must not be null.");
                 Debug.Assert(null != term, "The term must not be null.");
 
-                string queryTerm;
+                Term tm;
```

Comment:
Since you're creating new scopes for the cases of the switch below, there's no need to define this outside anymore. Otherwise, remove the extra scopes.

Reply:
I'm not sure if I understand this correctly. Even though the tm is initialized inside the scope, it is actually used outside the scope. Thats' why the definition is outside the switch case.